Soften language about single-use (#461)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-auth-scheme.md

({{!RFC9110, Section 11}}), with the scheme "PrivateToken". Origins challenge
clients to present a token from a specific issuer ({{challenge}}). Once a
client has received a token from that issuer, or already has a valid token
available, it presents the token to the origin ({{redemption}}). The process of
presenting a token in response to a challenge is also referred to as "spending" a token.

In order to prevent linkability across different transactions, clients
will often present a particular "PrivateToken" only once. Origins can link multiple
transactions to the same client if that client spends the same token value more

> than once. In deployment scenarios where origins send token challenges
> to request tokens, origins ought to expect at most one request containing a
> token from the client in reaction to a particular challenge.

Author: ```suggestion
than once. As such, origins ought to expect at most one unique token
value, carried in one request, for each challenge.
```

client has received a token from that issuer, or already has a valid token

> available, it presents the token to the origin ({{redemption}}).

Reviewer: ```suggestion
available, it presents the token to the origin ({{redemption}}). The process of
presenting a token in response to a challenge is also referred to as "spending" a token.
```

In order to prevent linkability across different transactions, clients

> will often present a particular "PrivateToken" only once. Spending the same
> token value more than once allows the origin to link multiple transactions to
> the same client. In deployment scenarios where origins send token challenges

Reviewer: ```suggestion
will often present a particular "PrivateToken" only once. Origins can link multiple
transactions to the same client if that client spends the same token value more
than once. In deployment scenarios where origins send token challenges
```

Reviewer: Less passive, more active

> the same client. In deployment scenarios where origins send token challenges
> to request tokens, origins ought to expect at most one request containing a
> token from the client in reaction to a particular challenge.

Reviewer: ```suggestion
the same client. As such, origins ought to expect at most one unique token
value, carried in one request, for each challenge.
```

This is also true in cases where the client isn't challenged. The suggestion also places emphasis on the uniqueness of the token, rather than of the request, since that's what this whole paragraph is about.

Author: Applied in a new commit
